recipes-core/images/riscv-nezha-image.bb

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION = "Basic image for nezha machine"
```

Comment:
Please move the change to replace `rng-tools` with `haveged` to nezha machine config. We do not want to create machine specific images if we dont have to. Because it will make other core reference images fail  but only this image work when using nezha

Reply:
I did not notice that since I have no board and I have no details about the machine itself. I think the best solution is to revert back the nezha image back to the kas file, check https://github.com/riscv/meta-riscv/pull/433/commits/e5b798bf60adaa8966703417a58f490ba40dc72f